Backpack-SwiftUI/AppSearchModal/Classes/BPKAppSearchModalState.swift

```diff
@@ -23,16 +23,5 @@ extension BPKAppSearchModal {
     public enum TextFieldState {
```

Comment:
lets remove this entire enum, it no longer makes sense

Reply:
Yes, I agree, I'm going to